src/io/flutter/run/daemon/DaemonConsoleView.java

```diff
@@ -81,12 +81,6 @@ public void print(@NotNull String text, @NotNull ConsoleViewContentType contentT
       return;
     }
 
-    // "flutter test" prints this when using --start-paused.
-    // (We should probably change it to stop printing it in machine mode.)
-    if (trimmed.startsWith("You should first set appropriate breakpoints, then resume the test in the debugger.")) {
-      return;
-    }
-
```

Comment:
Okay, it now checks the version number and only provides minimal support for testing on the alpha branch. Verified by switching back and forth. (The debug icon is enabled and disabled appropriately.)

The code assumes 0.0.11 supports `--machine` mode and 0.0.10 does not. That's currently true at HEAD on both branches.